From e5fbc7fd1e18cc23f6b2e906a622fc4452b70d0c Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 12 Jan 2025 11:11:07 -0500
Subject: [PATCH] Fix resolution open OpenGL library

---
 OpenGL/platform/glx.py | 21 +++++++++++++--------
 1 file changed, 13 insertions(+), 8 deletions(-)

diff --git a/OpenGL/platform/glx.py b/OpenGL/platform/glx.py
index 3fd684a..fcadd9f 100644
--- a/OpenGL/platform/glx.py
+++ b/OpenGL/platform/glx.py
@@ -12,17 +12,22 @@ class GLXPlatform(baseplatform.BasePlatform):
     @baseplatform.lazy_property
     def GL(self):
         try:
-            return ctypesloader.loadLibrary(
+            lib = ctypesloader.loadLibrary(
                 ctypes.cdll, "OpenGL", mode=ctypes.RTLD_GLOBAL
             )
         except OSError as err:
-            try:
-                # libGL was the original name, older devices likely still need it...
-                return ctypesloader.loadLibrary(
-                    ctypes.cdll, "GL", mode=ctypes.RTLD_GLOBAL
-                )
-            except OSError as err:
-                raise ImportError("Unable to load OpenGL library", *err.args)
+            lib = None
+
+        if lib is not None:
+            return lib
+
+        try:
+            # libGL was the original name, older devices likely still need it...
+            return ctypesloader.loadLibrary(
+                ctypes.cdll, "GL", mode=ctypes.RTLD_GLOBAL
+            )
+        except OSError as err:
+            raise ImportError("Unable to load OpenGL library", *err.args)
 
     @baseplatform.lazy_property
     def GLU(self):
-- 
2.47.1

