Index: OpenGL/GL/VERSION/GL_1_5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- OpenGL/GL/VERSION/GL_1_5.py	(revision bf11e7e59bff4a57dffbe79e1b27dac5a09166fc)
+++ OpenGL/GL/VERSION/GL_1_5.py	(revision abb9d9f2c7fae4d5b700e5f70e2ada62eb9211f9)
@@ -124,9 +124,6 @@
         size = ArrayDatatype.arrayByteCount( data )
     return baseOperation( target, offset, size, data )
 
-glGetBufferParameteriv = wrapper.wrapper(glGetBufferParameteriv).setOutput(
-    "params",(1,), orPassIn=True
-)
 @_lazy( glGetBufferPointerv )
 def glGetBufferPointerv( baseOperation, target, pname, params=None ):
     """Retrieve a ctypes pointer to buffer's data"""
